### PR TITLE
Redirect path for ephemeral tests

### PR DESCRIPTION
**Pull request by g-duval**

### CODE REVIEW

File: src/test/java/com/redhat/cloud/notifications/GwResourceTest.java [hunk 1]
```diff
@@ -175,8 +177,9 @@ public void shouldReturnServiceUnavailableForNonBadRequest() {
         }
     }
 
-    @Test
-    public void testNotificationsEndpoint() {
+    @ParameterizedTest
+    @ValueSource(strings = {"/notifications/", "/api/notifications-gw/notifications"}) // to test cases when endpoint have already some servers error or not
```
Comment (gwenneg) [inline src/test/java/com/redhat/cloud/notifications/GwResourceTest.java:181]: The comment looks wrong.

Comment (g-duval) [reply src/test/java/com/redhat/cloud/notifications/GwResourceTest.java]: that's right, thank you

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**